docs/operator-docs/create-rule-in-cumulus.md

```diff
@@ -0,0 +1,37 @@
+---
+id: create-rule-in-cumulus
+title: Create Rule In Cumulus
+hide_title: true
+---
+
+# Create Rule In Cumulus
```

Comment:
I think this section would be great with some screenshots.

Reply:
I might need someone with data and a real stack to take these for me. Thanks.

Reply:
@laurenfrederick Need assistance with this. Thanks.

Reply:
This should be fine to do with a test stack.

Reply:
Okay, I'll pair with someone on this since I don't have permissions.

Reply:
@laurenfrederick Able to pair with Brian and updated content with screenshots.

Reply:
Liking this a lot. Thanks!